librz/core/cmd/cmd_analysis.c

```diff
@@ -2742,6 +2673,166 @@ static bool cmd_aea(RzCore *core, int mode, ut64 addr, int length) {
 	return true;
 }
 
+// aeC
+RZ_IPI RzCmdStatus rz_analysis_appcall_handler(RzCore *core, int argc, const char **argv) {
+	for (int i = 1; i < argc; ++i) {
+		const char *alias = sdb_fmt("A%d", i - 1);
```

Comment:
avoid using `sdb_fmt`, either use `rz_strf` with a buffer on the stack or `rz_str_newf`